composer.json

```diff
@@ -14,6 +14,7 @@
         "composer/installers": "^1.0.20",
         "cweagans/composer-patches": "^1.0",
         "drupal-composer/drupal-scaffold": "^2.0.1",
+        "drupal/authorization": "1.x-dev",
```

Comment:
Does the solution require adding this as an explicit dependency? If not, we can let the LDAP module project specify the version that the need, which may avoid a conflict in the future.

Reply:
Updated to remove the drupal/authorization dependency.